### ISSUE TITLE
More details about evalution

### ISSUE THREAD

**Issue by YueWuHKUST** [crockwell/Cap3D#5]
Thanks for your great work!

Could you provide more details about how you generate test images using Point-E and Shap-E ? https://github.com/crockwell/Cap3D/blob/main/text-to-3D/README.md On this page, you says it will coming soon, but haven't beed updated.

Thanks

**Comment by tiangeluo** [comment]
Hi @YueWuHKUST ,

Thank you for your attention.

I've just updated the pointE inference and rendering codes. They were successfully tested in my env. Please let me know if you encounter any errors. 

ShapE code should come out tomorrow.

**Comment by tiangeluo** [comment]
Hey @YueWuHKUST ,

The shapE inference code is uploaded at https://github.com/crockwell/Cap3D/tree/main/text-to-3D/shap-e. I've tested this code in my environment. 

Please let me know if you face any errors or need further assistance.

**Comment by YueWuHKUST** [comment]
Hi tiange,

I successfully run your code, and there is a misleading part,

https://github.com/crockwell/Cap3D/tree/main/text-to-3D/point-e 
On this page, In step 4, for rendeing, the test_type is '300'. which should be 2k correct? 

According to paper, you use 2k uids for testing point-e

**Comment by tiangeluo** [comment]
nice! Oh, that point indeed may confuse people. I've updated it. Thanks for ur feedback.

**Comment by YueWuHKUST** [comment]
I have another question about your finetuning, did you follow your default epoch number in your code? You train point-e and shap-e for 25 epoch, and use the models at 25 epoch for evaluation? I'm trying to retrain your finetune model.

**Comment by tiangeluo** [comment]
Appendix I includes the fine-tuning details of the models used in the paper.

Due to computational constraints, we fine-tuned pointE and shapE for a total of 3 days each using 4 A40 GPUs.

<img width="851" alt="Screenshot 2023-09-06 at 2 37 16 PM" src="https://github.com/crockwell/Cap3D/assets/11552657/a6195bc0-a274-4d96-b26a-4e7d6ad202d4">

**Comment by YueWuHKUST** [comment]
yes, I read your appendix, Since I'm using V100 GPUs not A40. It is difficult for me to estimate a approximate epoch number of your training. Could you provide a approximate number of epoch number that you use to train for 3 days?

**Comment by tiangeluo** [comment]
Oh I see. The specific information are listed below: 

For pointE, we used epoch 12, iteration 3999 and batch size 256;
For shapE, we used epoch 3, iteration 19996 and batch size 64;

Note that we did not report the checkpoint with best score in validation set, but use the checkpoint runs till 3 days.

**Comment by tiangeluo** [comment]
Btw, @YueWuHKUST , we used a 330k training set for finetuning in our paper. This split is just uploaded to our dataset page: [train_330k.pkl](https://huggingface.co/datasets/tiange/Cap3D/blob/main/text-to-3D_test/train_330k.pkl). 

Please feel free to let me know if you need any other information.

**Comment by YueWuHKUST** [comment]
Oh, I thought you used https://github.com/crockwell/Cap3D/blob/main/text-to-3D/example_material/training_set.pkl for training in the finetuning, and use https://github.com/crockwell/Cap3D/blob/main/text-to-3D/example_material/validation_set.pkl for testing. Is the training_set.pkl is different from train_330.pkl?
